.github/settings.yml

```diff
@@ -103,7 +99,7 @@ labels:
 - name: component/vscode-extension
   color: 25399e
   description: MS Visual Studio Code OCM Extension
-
+## The different kinds of issues and pull requests - will be used for release notes grouping
 - name: kind/bug
```

Comment:
Why do we need bug and fix?

Reply:
Labels are used for both: issues and pull requests (which are a special type of issue)...

Can a PR be a 'bug'?
Can an issue be a 'fix'?

I'm open for suggestions and we can for sure agree on having only one... but which one? `bug_fix` :-)

Reply:
IIRC we only wanted to work with issues in the first place and PRs without issues should be an exception. That's why I would prefer it if we only provide this kind of label for the issues.

Reply:
That makes sense

Reply:
I was thinking of this: https://github.com/conventional-changelog/conventional-changelog

Reply:
i think going back to `kind/bugfix` and removing `kind/fix` and `kind/bug` sounds reasonable